hedera-node/hedera-mono-service/src/test/java/com/hedera/services/state/initialization/HfsSystemFilesManagerTest.java

```diff
@@ -99,7 +99,8 @@
 class HfsSystemFilesManagerTest {
     private static final String R4_FEE_SCHEDULE_REPR_PATH =
             "src/test/resources/testfiles/r4FeeSchedule.bin";
-    private static final String bootstrapJutilPropsLoc = "src/test/resources/bootstrap.properties";
+    private static final String bootstrapJutilPropsLoc =
+            "src/test/resources/bootstrap/hfsSystemFilesBootstrap.properties";
```

Comment:
This is intentional ?

Reply:
Yes, it was conflicting with the new `bootstrap.properties` file I added that fixed the `ServicesState` tests